client/db_test.go

```diff
@@ -245,3 +245,29 @@ func ExampleDB_Insecure() {
 	// Output:
 	// aa=1
 }
+
+func ExampleDB_Open() {
```

Comment:
This isn't a great "example", so I'd probably make it a normal `testing.T` test and take it out of the docs.